packages/server/src/main/java/software/uncharted/terarium/hmiserver/models/dataservice/project/Project.java

```diff
@@ -140,11 +140,11 @@ public String getEmbeddingSourceText() {
 			final ObjectMapper objectMapper = new ObjectMapper();
 			return objectMapper.writeValueAsString(this);
 		} catch (final Exception e) {
-			throw new RuntimeException("Failed to serialize model embedding text into JSON", e);
+			throw new RuntimeException("Failed to serialize project embedding text into JSON", e);
 		}
 	}
 
-	private String getOverviewAsReadableString() {
+	public String getOverviewAsReadableString() {
```

Comment:
You'll need to have `@TSIgnore` on this to have it not show up in the generated types